lib/screen/book_editor.dart

```diff
@@ -28,6 +28,8 @@ class _BookEditorState extends State<BookEditor> {
 
   final Book _book;
 
+  bool _validURL = false;
```

Comment:
Why do we need this variable?

Reply:
Euh as I am using the same view to save the book, I want to avoid renaming or saving the book to firebase storga, in case the book cover link in from the api